manifests/logging-stack/input-systemd.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: logging.kubesphere.io/v1alpha2
+kind: Input
+metadata:
+  name: kubelet
+  namespace: kubesphere-logging-system
+  labels:
+    logging.kubesphere.io/enabled: "true"
+    logging.kubesphere.io/component: logging
+spec:
+  systemd:
+    tag: service.kubelet
+    path: /var/log/journal
+    db: /fluent-bit/tail/kubelet.db
+    dbSync: Normal
+    systemdFilter:
+    - _SYSTEMD_UNIT=kubelet.service
+
+---
+apiVersion: logging.kubesphere.io/v1alpha2
+kind: Input
```

Comment:
docker is going to be replaced by other container runtimes like containerd or cri-o, so I think we can have 2 seperated inputs here which are input-kubelet.yaml and input-docker.yaml . Then user or installer will have an option to not installing the docker one